design/src/contributing/writing_and_debugging_a_low-level_feature_that_relies_on_HTTP.md

```diff
@@ -73,7 +73,7 @@ a request body for `aws-chunked` requires us to know the length of each chunk we
 have to prefix each chunk with its size in bytes, represented by one or more [hexadecimal] digits. To close the body, we
 send a final chunk with a zero. For example, the body "Hello world" would look like this when encoded:
 
-```
+```markdown
```

Comment:
Nit: Not sure `markdown` makes sense for this, maybe `console`?

Reply:
`txt`